package.json

```diff
@@ -196,6 +203,14 @@
     "zen-observable": "^0.8.15",
     "zod": "^3.22.2"
   },
+  "peerDependencies": {
+    "@chromaui/test-archiver": "0.0.53"
+  },
+  "peerDependenciesMeta": {
+    "@chromaui/test-archiver": {
+      "optional": true
+    }
+  },
```

Comment:
This was a pre-cursor to the new packages. We can tweak this to be those once ready.